blueocean-dashboard/src/main/js/PipelineRoutes.jsx

```diff
@@ -128,6 +126,27 @@ function onLeaveCheckBackground() {
 
 const trends = AppConfig.isFeatureEnabled('trends');
 
+class NotFoundAction extends React.Component {
```

Comment:
:ant:  For such component (simple presentation) the preferred form is: 

```javascript
const NotFoundAction = () => <NotFound/>;
```

Reply:
Actually why do you wrap `<NotFound/>` anyway?

Reply:
It fails providing the `NotFound` function to the router for some reason, it needs to be a class.